docs/navigate.md

```diff
@@ -32,7 +32,7 @@ The plugin supports these options:
         enableTouch: true
     }
 
-    propagateOriginTouch: false,
+    propagateSupportedGesture: false,
```

Comment:
I renamed it now since I have not got your agreement to keep it `propagateOriginTouch`.